Bugzilla/Template.pm

```diff
@@ -199,6 +199,13 @@ sub quoteUrls {
                ("\x{FDD2}" . ($count-1) . "\x{FDD3}")
               ~egox;
 
+    my $urlbase = Bugzilla->params->{'urlbase'};
+    $text =~ s~\bhttps?\Q://bugzil.la/\E([0-9]+)(\#c([0-9]+))?\b
+              ~($things[$count++] = $bug_link_func->($1, "${urlbase}show_bug.cgi?id=$1$2",
```

Comment:
maybe we should rewrite to `${urlbase}/$1$2` instead to encourage the native short bug url?

Reply:
oh - `${urlbase}/$1$2` currently doesn't get tooltip love in comments; that should be fixed too.

Reply:
I think this is out of scope for this change set and should be a separate bug.